controller/ingress_entry.go

```diff
@@ -64,6 +66,22 @@ func (e IngressEntry) validate() error {
 	if e.ServicePort == 0 {
 		return errors.New("missing service port")
 	}
+
+	var invalidAllowEntries []string
+	for _, allowEntry := range e.Allow {
+		if net.ParseIP(allowEntry) == nil {
+			if _, _, err := net.ParseCIDR(allowEntry); err != nil {
+				invalidAllowEntries = append(invalidAllowEntries, allowEntry)
+			}
+		}
+	}
+
+	if len(invalidAllowEntries) == 1 {
+		return fmt.Errorf("invalid entry in sky.uk/allow: %s", invalidAllowEntries[0])
```

Comment:
Any reason to have this additional block other than to change the message from "entry" to "entries"? `strings.Join` won't print the comma if there is only one element

Reply:
Yeah, just being picky grammatically, but there isn't much need for that. I'll simplify the code.